apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -110,6 +118,37 @@ private SpannerTestDatabase(string projectId)
             }
         }
 
+        private void CreateInstanceOnEmulatorIfNeeded(string projectId)
```

Comment:
It's the informal convention we use:
```suggestion
        private void MaybeCreateInstanceOnEmulator(string projectId)
```

Reply:
Done